pkg/resources/deployments/log-search-api.go

```diff
@@ -97,6 +100,19 @@ func logSearchAPIMeta(t *miniov1.Tenant) metav1.ObjectMeta {
 	for k, v := range t.LogSearchAPIPodLabels() {
 		meta.Labels[k] = v
 	}
+
+	// attach any labels
+	for k, v := range t.Spec.Log.Labels {
+		meta.Labels[k] = v
+	}
+	// attach any annotations
+	if len(t.Spec.Log.Labels) > 0 {
```

Comment:
shouldn't this be `if len(t.Spec.Log.Annotations) > 0 {`

Reply:
nice catch